charts/postgresql/README.md

```diff
@@ -12,7 +12,7 @@ $ helm install stable/postgresql
 
 This chart bootstraps a [PostgreSQL](https://github.com/bitnami/bitnami-docker-postgresql) deployment on a [Kubernetes](http://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager.
 
-Bitnami charts can be used with [Kubeapps](https://kubeapps.com/) for deployment and management of Helm Charts in clusters. This chart has been tested to work with NGINX Ingress, cert-manager, fluentd and Prometheus on top of the [BKPR](https://kubeprod.io/).
+Bitnami charts can be used with [Kubeapps](https://kubeapps.com/) for deployment and management of Helm Charts in clusters. This chart has been tested to work with NGINX Ingress, cert-manager, Vector and Prometheus on top of the [BKPR](https://kubeprod.io/).
```

Comment:
This statement is not accurate. The Bitnami Postgres chart has not been tested to work with vector. We should delete this whole line.